Composer/packages/client/src/pages/design/DesignPage.tsx

```diff
@@ -708,6 +810,37 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
             onDismiss={() => dismissManifestModal(projectId)}
           />
         )}
+        {brokenSkillInfo && (
+          <RepairSkillModal
+            skillItem={brokenSkillInfo}
+            onDismiss={() => {
+              setBrokenSkillInfo(undefined);
```

Comment:
setBrokenSkillInfo(); is the same as passing undefined to it

Reply:
setBrokenSkillInfo(); It would be an type error, if it's defined as <undefined | string>